configuration/backend_configuration/ampathforms/procedure-history-form.json

```diff
@@ -0,0 +1,225 @@
+                  }
+                ]
+              },
+              "id": "procedureOutcome",
+              "label": "Procedure outcome"
+            },
+            {
+              "type": "obs",
+              "questionOptions": {
+                "rendering": "number",
+                "concept": "165929AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                "min": "1",
+                "max": "100000",
+                "step": 1
+              },
+              "id": "procedureDuration",
+              "label": "Procedure duration"
+            },
+            {
+              "type": "obs",
+              "id": "procedureDurationUnit",
+              "label": "Duration unit",
+              "questionOptions": {
+                "rendering": "select",
+                "concept": "1732AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                "answers": [
+                  {
+                    "concept": "162582AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Number of occurrences"
+                  },
+                  {
+                    "concept": "162583AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Seconds"
+                  },
+                  {
+                    "concept": "1733AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Minutes"
+                  },
+                  {
+                    "concept": "1822AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Hours"
+                  },
+                  {
+                    "concept": "1072AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Days"
+                  },
+                  {
+                    "concept": "1073AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Weeks"
+                  },
+                  {
+                    "concept": "1074AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Months"
+                  },
+                  {
+                    "concept": "1734AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Years"
+                  }
+                ]
+              },
+              "clearWhenExpression": "Number(procedureDuration) > 99 && (procedureDurationUnit === '1734AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA')",
+              "validators": [
+                {
+                  "type": "js_expression",
+                  "failsWhenExpression": "procedureDurationUnit === '1734AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA' && Number(procedureDuration) > 99",
+                  "message": "Duration cannot exceed 100 years.. Change unit or reduce duration."
+                },
+                {
+                  "type": "js_expression",
+                  "failsWhenExpression": "procedureDurationUnit === '1733AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA' && Number(procedureDuration) > 1440",
+                  "message": "Duration cannot exceed 1440 minutes.. Change unit or reduce duration."
+                },
```

Comment:
I'd suggest we'd want this to be more like 2 days (2880 minutes) rather than 24 hours, unless there's a way to input something like 24 hours + 30 minutes or the like.